### PR TITLE
new indices for user, ruser, revision

### PR DESCRIPTION
**Pull request by jywarren**
Note these db indices - @icarito -- thanks!

* [x] all tests pass -- `rake test:all`

### CODE REVIEW

File: db/schema.rb.example [hunk 1]
```diff
@@ -190,10 +190,10 @@ ActiveRecord::Schema.define(:version => 20170824172336) do
   add_index "impressions", ["controller_name", "action_name", "request_hash"], :name => "controlleraction_request_index"
   add_index "impressions", ["controller_name", "action_name", "session_hash"], :name => "controlleraction_session_index"
   add_index "impressions", ["impressionable_type", "impressionable_id", "ip_address"], :name => "poly_ip_index"
-  add_index "impressions", ["impressionable_type", "impressionable_id", "params"], :name => "poly_params_request_index", :unique => false, :length => { "params" => 255 }
+  add_index "impressions", ["impressionable_type", "impressionable_id", "params"], :name => "poly_params_request_index"
```
Comment (jywarren) [inline db/schema.rb.example]: These impressions lines always get added in by my local; needs removal. That may fix this.

**Comment by jywarren** [comment]
Ok @icarito, I'm hoping these changes will fix this PR.

**Comment by jywarren** [comment]
Great, that solved it.

**Comment by icarito** [comment]
great, thanks!

**Pull request merged**